### ISSUE TITLE
Swipe left, right, up, down to dismiss

### ISSUE THREAD

**Issue by mschonvogel** [SCENEE/FloatingPanel#403]
I was wondering if you could provide an example of how to remove the panel by swipe left, right, up, and down. That'd be very helpful!

**Comment by SCENEE** [comment]
Have you tried `FloatingPanelController.isRemovalInteractionEnabled`?

**Comment by mschonvogel** [comment]
Yes @SCENEE, but this only enables a dismiss by the swipe-down gesture. How can I dismiss the panel by swiping left/right?

**Comment by SCENEE** [comment]
Alright, @michalraska. You can layout a panel positioned at top, left and right by specifying `FloatingPanelLayout.position` and then are able to remove the panel by swipe up, left, or right 👍

If you use a bottom positioned panel, you need to implement a swipe gesture to dismiss the panel by swiping left/right on the surface view or the root view(FloatingPanelContorller.view) according to your panel's content view controller. [`FloatingPanelPanGestureRecognizer.delegateProxy`](https://github.com/SCENEE/FloatingPanel/blob/master/Documentation/FloatingPanel%202.0%20Migration%20Guide.md#floatingpanelpangesturerecognizer) might help you 😌
